dev/snippets/config/skeletons/sample.html

```diff
@@ -14,7 +14,7 @@
   <div class="copyable-container">
     <button class="copy-button-overlay copy-button" title="Copy to clipboard"
       onclick="copyTextToClipboard(longSnippet{{serial}});">
-      <i class="material-icons copy-image">assignment</i>
+      <i class="material-icons copy-image">content_copy</i>
```

Comment:
Seems this and the change below is not related to this PR. And looks like this change has been merged to master:)

Reply:
Oh!  Thanks, yes, that's not related.